### PR TITLE
Hello World

### PR DESCRIPTION
**Pull request by Lexicality**
I had no reason to do this right now but I will in the future

### CODE REVIEW

File: prometheus_django/middleware.py [hunk 1]
```diff
@@ -12,53 +14,57 @@
...
-                                       'Requests in progress',
-                                       ['method', 'endpoint'])
+PROMETHEUS_REQUEST_DURATION = Summary(
+    "http_request_duration_microseconds",
+    "Request duration in microseconds",
+    ["method", "endpoint"],
+)
+PROMETHEUS_REQUEST_TOTAL = Counter(
+    "http_requests_total",
+    "Request total wiht status",
```
Comment (marekzp) [inline prometheus_django/middleware.py:33]: I'll just add this why I remember, there is a typo in the word with

**Review by marekzp — COMMENTED** [review]
Before we go too far down this route, I would like to discuss what metrics we want to get from Prometheus that we are not getting via sentry.

**Comment by charlie-bloom** [comment]
Hey guys,

Old lantumite here, i'm getting notifications on this thread just fyi.
Might need to remove me from that github account or something like that.

charlie-bloom is my github account

Hope it's all going well there!

**Comment by Lexicality** [comment]
> Before we go too far down this route, I would like to discuss what metrics we want to get from Prometheus that we are not getting via sentry.

Let's have a quick chat after standup but essentially with Prometheus we get realtime graphs with whatever things we want to annotate while Sentry will give us aggregated metrics after the fact



> Hey guys, Old lantumite here, i'm getting notifications on this thread just fyi. Might need to remove me from that github account or something like that. charlie-bloom is my github account Hope it's all going well there!

👋 aloha, going fairly well this year! You're getting notifications because this is a public repo (yay open source!) and github loves to subscribe you to things and never unsubscribe so you're probably still watching all our other public repos too

File: prometheus_django/middleware.py (hunk 1 above)
Comment (Lexicality) [reply prometheus_django/middleware.py:33]: I don't actually know what happens with Prometheus if you modify an already existing counter definition...

**Review by marekzp — COMMENTED** [review]
I'd put the first commit into a separate PR. Again, the key question is which services use the master branch and how are we going to test for whether they build and run successfully?

**Review by marekzp — APPROVED** [review]

**Pull request merged**